### PR TITLE
Tips UI

### PR DESCRIPTION
**Pull request by syc0n**
This will popup once you log in to the server, just after the loading screen. The popup will remain visible for 15 seconds, fade out, and close itself. A player can click "Keep open" to keep the Tips UI open.
A player can click the 'X' to close the Tips UI

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/CUModSquad/cu-ui/pull/1" target="_blank">ReviewNinja

File: tips/tips.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            <div class="window-bottom" align="center">
+                <div>
+                    <button id="btn-keepOpen">Keep open</button>
+                    <div  id="timer">Closing in: </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script src="../vendor/jquery-2.1.1.js"></script>
+    <script src="../vendor/jquery-ui.min.js"></script>
```
Comment (codecorsair) [inline tips/tips.html]: You're not using jquery-ui here, so you can remove this. !fix

File: tips/tips.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <script src="../vendor/lodash.js"></script>
```
Comment (codecorsair) [inline tips/tips.html]: Also not using lodash, you can remove this as well. !fix

Comment (syc0n) [inline tips/tips.html]: !resolved

Comment (syc0n) [inline tips/tips.html]: !fix

**Comment by codecorsair** [comment]
also, before you submit a pull request, please run an interactive rebase and squash all the intermediary commits.

File: tips/tips.html (hunk 2 above)
Comment (codecorsair) [inline tips/tips.html]: !fixed

File: tips/tips.html (hunk 1 above)
Comment (codecorsair) [inline tips/tips.html]: !fixed

**Pull request closed without merging**